{"uuid":"e9577d24-02c7-4b8b-aa5a-d3915c9a729f","name":"Scenario Outline: The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","children":["d374f7e6-f4c0-4760-aa34-2dd1a3ce66fd"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"e7df2931-b85e-46fd-b93b-99cf6f71d106-attachment","type":"image/png"},{"name":"Myscreenshot","source":"110c3735-93ba-40e1-9cff-5012b60a6294-attachment"}],"parameters":[],"start":1696884711169,"stop":1696884711364},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884711365,"stop":1696884711365},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884711366,"stop":1696884711366},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884711367,"stop":1696884711367}],"start":1696884711165,"stop":1696884711367}